rfc/rfc-0019-rfc-process.md

```diff
@@ -0,0 +1,112 @@
+
+# How it Works
+[how-it-works]: #how-it-works
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the community and the maintainer team.
+
+## What's in Scope
+
+You'll need to follow this process for anything considered "substantial". The things that we consider "substantial" will evolve over time, but will generally be user facing changes.
+
+What constitutes a "substantial" change may include the following but is not limited to:
+
+- changes to a resource's spec
+- changes to a resource's status
+- breaking changes
+- governance changes
+- issues that need more discussion or clarification as determined by the project maintainers
+
+If you submit a pull request or issue that the team deems warrants an RFC, it will be politely closed with a request for an RFC.
+
+## Process
+
+### RFCs
+
+To get an RFC into Cartographer, first the RFC needs to be merged into the repo. Once an RFC is merged, it's considered 'active' and may be implemented to be included in the project. These steps will get an RFC to be considered:
+
+- Fork the repo: <https://github.com/vmware-tanzu/cartographer>
+- Copy `rfc/rfc-0000-template.md` to `rfc/rfc-0000-my-feature.md` (where 'my-feature' is descriptive. don't assign an RFC number yet).
+- Fill in RFC. Any section can be marked as "N/A" if not applicable.
+- Submit a pull request. The pull request is the time to get review of the proposal from the larger community.
+- Build consensus and integrate feedback. RFCs that have broad support are much more likely to make progress than those that don't receive any comments.
+
+Once a pull request is opened, the RFC is now in development and the following will happen:
+
+- It will be discussed in a future community meeting. Community meetings happen on a weekly cadence barring exceptions.
+- The team will discuss as much as possible in the RFC pull request directly. Any outside discussion will be summarized in the comment thread.
+- When deemed "ready", a team member will propose a "motion for final comment period (FCP)" along with a disposition of the outcome (merge, close, or postpone). This is step taken when enough discussion of the tradeoffs have taken place and the team is in a position to make a decision. Before entering FCP, a super majority of the team must sign off.
+- The FCP will last 7 days. If there's unanimous agreement among the team the FCP can close early.
+- For voting, the binding votes are comprised of the maintainer team. Acceptance requires a super majority of binding votes in favor. The voting options are the following: Affirmative, Negative, and Abstinence. Non-binding votes are of course welcome. Super majority means 2/3 or greater.
```

Comment:
Say 5 maintainers abstain, 2 vote Affirmative and 1 votes Negative.

Do the Affirmatives carry the day?